src/charm.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
+# pylint: disable=C,R
```

Comment:
We have a standard for explaining the disables, is it applicable here?

Reply:
Also applicable for all other linting disables

Reply:
Merged main.